examples/wallet_with_pincode/wallet_with_pincode.v.py

```diff
@@ -0,0 +1,77 @@
+# Simple wallet with PIN protection
+
+# bad guys can not get money out of this contract-wallet
+# even if your secret key has been compromised
+# if they do not know the `secret`
+# (pre-image of the hash that is stored in contract).
+
+# Owners wallet.
+owner: public(address)
+
+# Hash of a secret number that allows user
+# to operate with funds from this contract.
+secret_hash: public(bytes32)
+
+# Assign owner to contract creator.
+def __init__():
+    self.owner = msg.sender
+
+# This function allows to reset secret
+# each secret should be used only once
+# since everyone can track your transaction data
+# in blockchain.
+def set_secret(_secret: num, _new_secret: bytes32):
```

Comment:
Make `_new_secret` into `_new_secret_hash` so it is clearer what it's purpose is